docs/guides/integration-aws-entity-resolution.md

```diff
@@ -138,7 +139,7 @@ When your initial account creation and setup are done, the next step is to creat
 The following steps are a summary of actions you'll take in the AWS Entity Resolution user interface. For additional details and instructions, see [Creating a matching workflow with UID 2.0](https://docs.aws.amazon.com/entityresolution/latest/userguide/create-matching-workflow-provider.html#create-mw-uid) in the AWS Entity Resolution documentation.
 
 :::note
-UID2 supports either email or phone number for UID2 generation. However, in AWS Entity Resolution, if both values are present in the schema mapping, the workflow uses the email and treats the phone number as a pass-through field. If your data includes a mix of emails and phone numbers, the best approach is to create a separate workflow for each, with separate schema mappings. In this scenario, go through the following steps twice&#8212;create one workflow for emails and a separate one for phone numbers.
+UID2 supports both email and phone number for UID2 generation. However, in AWS Entity Resolution, if both values are present in the schema mapping, the workflow will duplicate each record in the output. One record will use the email for UID2 generation and the second record will use phone number. If you don't prefer this duplication of records in the output and if your data includes a mix of emails and phone numbers, the best approach is to create a separate workflow for each, with separate schema mappings. In this scenario, go through the following steps twice&#8212;create one workflow for emails and a separate one for phone numbers.
```

Comment:
UID2 supports both email and phone number for UID2 generation. However, in AWS Entity Resolution, if both values are present in the schema mapping, the workflow duplicates each record in the output. One record uses the email for UID2 generation and the second record uses phone number. If your data includes a mix of emails and phone numbers and you don't want this duplication of records in the output, the best approach is to create a separate workflow for each, with separate schema mappings. In this scenario, go through the following steps twice&#8212;create one workflow for emails and a separate one for phone numbers.

(just use present tense instead of future. Also switched the sequence of the two list items)